Makefile

```diff
@@ -0,0 +1,12 @@
+all: test
+
+clean:
+	find . -name '*.pyc' -exec rm \{\} \;
+
+flake8:
```

Comment:
Unnecessarily complex. Check out the [country template](https://github.com/openfisca/country-template/blob/master/Makefile#L5-L7) for a much simpler set up.